OpenEdXMobile/res/layout/activity_course_base.xml

```diff
@@ -1,190 +1,180 @@
-<android.support.v4.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/drawer_layout"
+    android:id="@+id/coordinator_layout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="@color/white"
     android:fitsSystemWindows="true"
     android:splitMotionEvents="false">
 
-    <android.support.design.widget.CoordinatorLayout
-        android:id="@+id/coordinator_layout"
+    <!--
+    Normally, we shouldn't need a separate LinearLayout under a CoordinatorLayout to hold the
+    toolbar and any content under it. But in this case its necessary cuz our tests start to
+    fail without any valid/explanatory reasons. (CourseUnitNavigationActivityTest &
+    CourseOutlineActivityTest)
+    TODO: Get rid of this layout when we remove the Navigation drawer from our app and convert CourseUnitNavigationActivity to use BaseSingleFragmentActivity as a parent. (i.e flag: TAB_LAYOUTS_ENABLED=FALSE)
+    -->
+    <LinearLayout
         android:layout_width="match_parent"
-        android:layout_height="match_parent">
-
-        <!--
-        Normally, we shouldn't need a separate LinearLayout under a CoordinatorLayout to hold the
-        toolbar and any content under it. But in this case its necessary cuz our tests start to
-        fail without any valid/explanatory reasons. (CourseUnitNavigationActivityTest &
-        CourseOutlineActivityTest)
-        TODO: Get rid of this layout when we remove the Navigation drawer from our app and convert CourseUnitNavigationActivity to use BaseSingleFragmentActivity as a parent. (i.e flag: TAB_LAYOUTS_ENABLED=FALSE)
-        -->
-        <LinearLayout
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:orientation="vertical">
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+
+        <include layout="@layout/toolbar" />
 
-            <include layout="@layout/toolbar" />
+        <FrameLayout
+            android:id="@+id/content_error_root"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_weight="1">
 
-            <FrameLayout
-                android:id="@+id/content_error_root"
+            <LinearLayout
+                android:id="@+id/content_area"
                 android:layout_width="match_parent"
-                android:layout_height="0dp"
-                android:layout_weight="1">
+                android:layout_height="match_parent"
+                android:orientation="vertical">
 
                 <LinearLayout
```

Comment:
This layout is not needed anymore. Its references in the CourseBaseActivity should also be removed.